pkgs/applications/version-management/gitoxide/default.nix

```
@@ -1,24 +1,29 @@
-{ lib, rustPlatform, fetchFromGitHub }:
+{ lib, rustPlatform, fetchFromGitHub, openssl, pkgconfig }:
```

```suggestion
{ lib, rustPlatform, fetchFromGitHub, pkg-config, openssl }:
```

`pkgconfig` is an alias for `pkg-config`. Also, we generally order `nativeBuildInputs` before `buildInputs`.